examples/gallery/lines/roads.py

```diff
@@ -24,10 +24,10 @@
 
 fig = pygmt.Figure()
 
-# Define target region around O'ahu (Hawai'i)
+# Define target region around Oʻahu (Hawaiʻi)
 region = [-158.3, -157.6, 21.2, 21.75]  # xmin, xmax, ymin, ymax
 
-title = r"Main roads of O\047ahu (Hawai\047i)"  # \047 is octal code for '
+title = r"Main roads of O\140ahu (Hawai\140i)"  # \140 is octal code for `, approximating the Okina ʻ
```

Comment:
If we're going to use backtick, then we don't need to use its octal code.
```suggestion
title = r"Main roads of O`ahu (Hawai`i)"  #  Approximating the Okina ʻ with `
```